### PR TITLE
Add support for RevPi Connect 4

### PR DESCRIPTION
**Pull request by nbuchwitz**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "F",   # Pyflakes
+    "Q",   # flake8-quotes
+]
+ignore = [
+    "ANN101", # missing-type-self
+    "D107",   # undocumented-public-init
+]
+
+[tool.ruff.mccabe]
+max-complexity = 10
```
Comment (RamiGspo) [inline pyproject.toml:17]: If possible, please add here a empty line...

**Comment by RamiGspo** [comment]
I really like the way you have used the conventional commits.... Really nice! :+1:  :rocket:

File: revpi_provisioning/cli/__main__.py
Comment (naruxde) [inline revpi_provisioning/cli/__main__.py:1]: Use this __main__ as template. It's important to add the path if __package__ is empty. I pickt this from other libraries like pip.

Comment (nbuchwitz) [reply revpi_provisioning/cli/__main__.py:1]: See [523f651](https://github.com/RevolutionPi/revpi-eol-provisioner/pull/14/commits/523f6516ed1481d2be40a96adb01577a9c0961b2)

**Pull request merged**